docker/airflow/dags/colombia_dag.py

```diff
@@ -0,0 +1,45 @@
+from airflow.decorators import dag, task
+from airflow.operators.empty import EmptyOperator
+import pendulum
+from datetime import timedelta
+
+DATA_PATH = "/opt/EpiGraphHub/data_collection"
+import sys
+
+sys.path.insert(0, DATA_PATH)
```

Comment:
we probably should need to move data_collection to epigraphhub_py ... so we will not need this workaround